bigbone/src/main/kotlin/social/bigbone/api/method/AnnouncementMethods.kt

```diff
@@ -0,0 +1,81 @@
+package social.bigbone.api.method
+
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.Parameters
+import social.bigbone.api.Pageable
+import social.bigbone.api.entity.Announcement
+import social.bigbone.api.exception.BigBoneRequestException
+
+/**
+ * Allows access to API methods with endpoints having an "api/vX/announcements" prefix.
+ * @see <a href="https://docs.joinmastodon.org/methods/notifications/">Mastodon notifications API methods</a>
```

Comment:
Copy-paste mistake? 😊 

```suggestion
 * @see <a href="https://docs.joinmastodon.org/methods/announcements/">Mastodon announcements API methods</a>
```

Reply:
Oops silly mistake 😅